docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+
+<br/>
+
+### Alternative assays
+
+The cellxgene data portal and explorer are able to handle a wide range of single cell data types. Due to requirements of the cellxgene schema and limitations of the `AnnData` object structure, there are some assay specific considerations that are outlined below
+
+- ATAC/mC
+  - raw data: Since there is not a standard way to generate a counts matrix or gene activity matrix for single cell epigenetic data, it is suitable to put a copy of `adata.X` in place of a raw data matrix. You can specify this assignment in `adata.uns`
+  - [(Not Live)Link to example curated dataset](www.example.com)
+- RNAseq
+  -  In some single cell toolchains, outputs of different computational methods may be of a different dimensionality than the input matrix (i.e SCTransform, or the scale.data slot in Seurat objects). Since `AnnData` objects do not allow for matrices of different dimensions in `adata.layers`, it is suitable to pad the missing rows/features with zeros to ensure that indices (feature names) across different the different layers are matched. You can specify that these matrices were padded in `adata.uns['layer_descriptions']` as a part of the cellxgene schema. This allows users who may reuse your data to remove the padding from the matrix before working with the data further.
+  -  [(Not Live)Link to example curated dataset](www.example.com)
+- CITEseq
+  - TODO: considerations for CITE-seq...
+  - [(Not Live)Link to example curated dataset](www.example.com)
+- Spatial/Visium
+  - Since spatial sequencing spots are not at the single cell level, you may wish to include prediction matrices that provide deconvolution information for each spot. These can generally be stored in `adata.uns` although it can be interesting to add prediction scores for each spot as new columns to the `adata.obs` dataframe
+  - [(Not Live)Link to example curated dataset](www.example.com)
+
+<br/>
+
+---
+
+## Schema definition
+
+<br/>
+
+The purpose of the cellxgene schema is to support the construction of a data corpus that facilitates data integration across multiple tissues and experiments. This goal requires that we collect a standardized set of metadata about single cell datasets that are uploaded to the cellxgene data portal. To make this process easy to adhere to, we only require a few fields (detailed below) to support easy search and integration across datasets. These metadata are stored within the `AnnData` object (in `adata.uns` and `adata.obs`). To access a more comprehensive decription about our schema requirements, you can refer to the [official schema definition](corpora_schema.md).
```

Comment:
_decription -> description_

Generally, I try to avoid "we", "ours", etc.

_This goal requires t~hat we collect~ a standardized set of metadata ~about~  for the single cell datasets that are uploaded to the cellxgene data portal. To make this process easy to adhere to, ~we~ the schema only requires a few fields (detailed below) to support easy search and integration across datasets._